composer.json

```diff
@@ -23,7 +24,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Google\\": "tests"
+            "Google\\ApiCore\\UnitTests\\": "tests"
```

Comment:
As part of the most recent restructuring (moving to ApiCore) I switched the test directory structure/namespace to use `Tests/Unit`, to mirror what we use in google-cloud-php. Is the change back to using `UnitTests` intentional? I am happy to go back to this if we think it is better.

Reply:
I like `Tests\Unit`, I must have goofed with `UnitTests` on the rebase. Apologies.

Reply:
Wouldn't this structure make more sense:

```
tests/Unit/ApiCore/LongRunning/OperationsClientTest.php
tests/Unit/ApiCore/Middleware/RetryMiddlewareTest.php
tests/Unit/ApiCore/testdata/...
tests/Unit/ApiCore/*Test.php
```

```json
"autoload-dev": {
    "psr-4": {
        "Google\\Tests\\": "tests"
    }
}
```

with the namespaces then being  `Google\Tests\Unit\ApiCore` and `Google\Tests\Unit\ApiCore\LongRunning`? This would be more consistent with `google-cloud-php`

Reply:
Would that be more consistent? Google Cloud PHP puts tests under the `Google\Cloud` namespace, so I would have thought the corresponding namespace would be `Google\ApiCore`. (See here: https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/tests/unit/Datastore/BlobTest.php#L18)

Reply:
Ahh I see, yes so we would want `Google\ApiCore\Tests\Unit`. Hmm, this is okay but then why do we have `Google\ApiCore\LongRunning\Tests\Unit` instead of `Google\ApiCore\Tests\Unit\\LongRunning` ?

Reply:
Good point. I think we should change to `Google\ApiCore\Tests\Unit\LongRunning`